src/pongo/drivers/recfg/meson.build

```diff
@@ -0,0 +1,4 @@
+pongo_drivers_recfg_sources_c = [
+  'src/pongo/drivers/recfg/recfg.c',
```

Comment:
Consider using `files()` function and skipping the initial directory path component within the sources list.

Renaming and moving directories around would be so much easier by not having to change all the build files in a subtree.